cmd/connectivity_pod_to_pod.go

```diff
@@ -30,19 +29,19 @@ func newConnectivityPodToPodCmd() *cobra.Command {
 				return errors.Errorf("provide both SOURCE_POD and DESTINATION_POD")
 			}
 
-			fromPod, err := kuberneteshelper.PodFromString(args[0])
+			sourcePod, err := pod.FromString(args[0])
 			if err != nil {
 				return errors.New("invalid SOURCE_POD")
 			}
 
-			toPod, err := kuberneteshelper.PodFromString(args[1])
+			destinationPod, err := pod.FromString(args[1])
```

Comment:
For consistency w/ the rest of the repo, can we change `destinationPod` to `dstPod` ?